content/en/serverless/aws_lambda/distributed_tracing.md

```diff
@@ -108,15 +110,44 @@ For .NET serverless applications, Datadog recommends [installing Datadog's traci
 
 Learn more about [tracing through .NET Azure serverless applications][15].
 
-### Hybrid environments
+## Span Auto-linking
+Datadog automatically detects linked spans when segements of your asynchronous requests cannot propogate trace context, such as when a request triggers an [S3 Change Events][28], or [DynamoDB Streams][29]. These span links will appear in the [Span Links tab][30] in the Datadog Trace UI. 
+
+**Note:** Span auto-linking may not link traces if you are only ingesting a sample of your traces because the linked traces might be dropped before ingestion. To improve your chances of seeing auto-linked spans, increase your sample rate. 
+
+If you are viewing the request that originated before the change event and the linked trace is ingested, you can the linked span as a `Backward` link. 
+
+If you are viewing the request that originated before the change event and the linked trace is ingested, you can see the linked span as a `Forward` link. 
```

Comment:
Is this a link in the UI that says "Forward" or "Backward". Just want to make sure it's clear. Thanks!

Reply:
yeah it's a tag, it looks like this - 

<img width="915" alt="image" src="https://github.com/user-attachments/assets/fbff6c9a-00f5-41a9-9a2d-deb0ef5ea8b2">